public/results/index.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Seattle Flu Study</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../style.css">
+  <link rel="shortcut icon" href="../favicon.ico">
+</head>
+
+<body>
+  <header>
+    <nav class="home">
+        <menu>
+          <li>
+            <a href="/">Home</a>
+          </li>
+          <li>
+            <a href="/uw">Husky Coronavirus Testing</a>
+          </li>
+          <li>
+            <a href="https://scanpublichealth.org">SCAN</a>
+          </li>
+          <li>
+            <a href="/results">Results</a>
+          </li>
+        </menu>
+      </nav>
+    <h1>Seattle Flu Study</h1>
+    <img alt="Image with text that says 'powered by the' plus the Seattle Flu Study logo"
+      src="../images/logo_SFS.svg">
+  </header>
+
+    <h2>Access Your Results</h2>
+  <button
+    onclick="window.location.href='https://securelink.labmed.uw.edu/seattleflu'"">
+    Click here to access your test results
+  </button>
```

Comment:
My (vague) understanding is that using a <button> + onclick is unfriendly to accessibility software. The more conventional approach is to stick to an <a> but then style it like a button. (Frameworks like Bootstrap come pre-built with such styles.)

Reply:
Gotcha, that's good to know. I'll add a commit that improves accessibility.

Reply:
Addressed in a new commit.